spec/features/trainees/creating_a_new_trainee_spec.rb

```diff
@@ -21,20 +21,40 @@
     then_i_should_see_the_new_trainee_overview
   end
 
-  scenario "provider led (postgrad) radio button not shown when feature set to false", 'feature_routes.provider_led_postgrad': false do
-    and_i_should_not_see_provider_led_postgrad_route
-  end
-
   scenario "setting up an initial early years undergrad record", 'feature_routes.early_years_undergrad': true do
     and_i_select_early_years_undergrad_route
     and_i_save_the_form
     then_i_should_see_the_new_trainee_overview
   end
 
+  scenario "provider led postgrad radio button not shown when feature set to false", 'feature_routes.provider_led_postgrad': false do
+    and_i_should_not_see_provider_led_postgrad_route
+  end
+
   scenario "early years undergrad radio button not shown when feature set to false", 'feature_routes.early_years_undergrad': false do
     and_i_should_not_see_early_years_undergrad_route
   end
 
+  scenario "early years assessment only radio button not shown when feature set to false", 'feature_routes.early_years_assessment_only': false do
+    and_i_should_not_see_early_years_assessment_only_route
+  end
+
+  scenario "early years graduate entry radio button not shown when feature set to false", 'feature_routes.early_years_graduate_entry': false do
+    and_i_should_not_see_early_years_graduate_entry_route
+  end
+
+  scenario "early years graduate placement radio button not shown when feature set to false", 'feature_routes.early_years_graduate_placement': false do
+    and_i_should_not_see_early_years_graduate_placement_route
+  end
+
+  scenario "school direct salaried radio button not shown when feature set to false", 'feature_routes.school_direct_salaried': false do
+    and_i_should_not_see_school_direct_salaried_route
+  end
+
+  scenario "school direct tuition fee radio button not shown when feature set to false", 'feature_routes.school_direct_tuition_fee': false do
```

Comment:
As per my comment above, if you write the opposite test it would fail e.g.

```
scenario "school direct tuition fee radio button shown when feature set to true", 'feature_routes.school_direct_tuition_fee': true do
  and_i_should_see_school_direct_tuition_fee_route
end
```

=> would theoretically fail